db/migrate/20230509214723_separate_trusted_users_can_edit_others_site_setting.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class SeparateTrustedUsersCanEditOthersSiteSetting < ActiveRecord::Migration[7.0]
+  def up
+    if select_value("SELECT 1 FROM site_settings WHERE name = 'trusted_users_can_edit_others'")
+      execute <<~SQL
+        DELETE FROM site_settings WHERE name = 'trusted_users_can_edit_others';
+        INSERT INTO site_settings (name, data_type, value, created_at, updated_at) VALUES ('edit_all_topic_groups', 20, '', now(), now());
+        INSERT INTO site_settings (name, data_type, value, created_at, updated_at) VALUES ('edit_all_post_groups', 20, '', now(), now());
```

Comment:
Sorry @rngus2344 it just occurred to me that the default values in `site_settings.yml` is "13" and "14". Should that be the value here as well?

Can you write a test case for when `SiteSetting.edit_all_post_groups` is not empty please?

Reply:
@nattsw I believe this is because the default value previously was `true` for the site setting. So this is saying, if there is a site settings record for the old setting, delete it and create 2 new settings with empty values (no groups), to mirror the old behavior. Does that makes sense?

Reply:
```
if select_value("SELECT 1 FROM site_settings WHERE name = 'trusted_users_can_edit_others'")
```

this assumes the value will be `f`? The value in the db can be `t` too if the admin toggles it back I believe.

Reply:
ohhh wow @rngus2344 I really was convinced the rows wouldn't be present with `f` ever because it's the default but I guess I'm wrong on that. Should also check in the `select_value` for where the value is `f`.